buildscripts/checkdeps.sh

```diff
@@ -129,7 +129,9 @@ assert_check_golang_env() {
         exit 1
     fi
 
-    if [ -z "${GOPATH}" ]; then
+    # Since go 1.8 GOPATH is defaulted to ${HOME}/go, we don't need to validate GOPATH env anymore.
+    gopath=$(go env GOPATH)
+    if [ -z "${gopath}" ]; then
```

Comment:
Unable to understand the comment and this check.  Why do we need to do this?

Reply:
actually you are right this check is not needed anymore.